packages/bot-web-ui/src/stores/transactions-store.js

```diff
@@ -1,14 +1,28 @@
-import { action, observable } from 'mobx';
+import { action, observable, reaction } from 'mobx';
 import { formatDate } from '@deriv/shared';
 import { isEnded } from '@deriv/bot-skeleton';
 import { transaction_elements } from '../constants/transactions';
 
+const transaction_storage_key = 'transaction_cache';
+
 export default class TransactionsStore {
     constructor(root_store) {
         this.root_store = root_store;
+
+        reaction(
+            () => this.elements,
+            elements => sessionStorage.setItem(transaction_storage_key, JSON.stringify(elements))
+        );
     }
 
-    @observable elements = [];
+    getTransactionStorageKey = key => {
```

Comment:
Same here, try to name the function after what it's doing. We're not getting the key here.